assets/queries/terraform/alicloud/cmk_is_unusable/query.rego

```diff
@@ -8,6 +8,8 @@ CxPolicy[result] {
 
 	resource.is_enabled == false
 
+	remediation := {"before": "false", "after": "true"}
```

Comment:
A suggestion (for future versions) would be to refactor this JSON construction into a function, for instance:
`replace(A,B)`, whose output would be exactly `json.marshal({"before":A, "after":B})`
And for other cases
`addProperty(Prop,Val)`, whose output would be exactly `Prop: Val`

Or we can be even further and create a library of constant remediations...
`replaceTrueWithFalse()` that would return exactly `json.marshal({"before":"true", "after":"false"}`

This would reduce the repeated code throughout queries

Reply:
> A suggestion (for future versions) would be to refactor this JSON construction into a function, for instance: `replace(A,B)`, whose output would be exactly `json.marshal({"before":A, "after":B})` And for other cases `addProperty(Prop,Val)`, whose output would be exactly `Prop: Val`
> 
> Or we can be even further and create a library of constant remediations... `replaceTrueWithFalse()` that would return exactly `json.marshal({"before":"true", "after":"false"}`
> 
> This would reduce the repeated code throughout queries

Yes, I totally agree. Thank you so much for the suggestions.